modules/localstack/localstack_test.go

```diff
@@ -165,3 +175,60 @@ func TestStartWithNetwork(t *testing.T) {
 	require.Equal(t, 1, len(networks))
 	require.Equal(t, "localstack-network", networks[0])
 }
+
+func TestStartV2WithNetwork(t *testing.T) {
+	ctx := context.Background()
+
+	nw, err := testcontainers.GenericNetwork(ctx, testcontainers.GenericNetworkRequest{
+		NetworkRequest: testcontainers.NetworkRequest{
+			Name: "localstack-network",
+		},
+	})
+	require.Nil(t, err)
+	assert.NotNil(t, nw)
+
+	localstack, err := StartContainer(
+		ctx,
+		OverrideContainerRequest(testcontainers.ContainerRequest{
+			Image:          "localstack/localstack:latest",
```

Comment:
Replace by proper tag. It should be `v2.0`

```suggestion
			Image:          "localstack/localstack:v2.0",
```